narwhals/_pandas_like/series.py

```diff
@@ -504,6 +504,9 @@ def ends_with(self, suffix: str) -> PandasSeries:
             self._series._series.str.endswith(suffix),
         )
 
+    def contains(self, pat: str, regex: bool = True) -> PandasSeries:
+        return self._series._from_series(self._series._series.str.contains(pat, regex))
```

Comment:
This is possibly an opinionated comment, but rather than mapping parameters dynamically in `narwhals/expression.py`, I would do it in here. Namely:
```suggestion
    def contains(self, pattern: str, literal: bool) -> PandasSeries:
        return self._series._from_series(self._series._series.str.contains(pat=pattern, regex=not literal))
```

With that in place, the other implementations should follow almost seamlessly by reusing this one directly.
What do you think?

Reply:
Also, `regex = ...` should be explicitly specified, as otherwise the regex variable will be passed to the second argument of `str.contains` which is `case` and not `regex` itself ([docs](https://pandas.pydata.org/docs/reference/api/pandas.Series.str.contains.html))

Reply:
Yeah, this is much cleaner